### ISSUE TITLE
An error shows in the "Logs" node for one basic Function App slot

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-azurefunctions#3423]
**OS:** Win10
**Build Version:** 20221109.16

**Repro Steps:**
1. Create a basic Function App.
2. Create one slot for this Function App.
3. Expand the "Logs" node of this slot.
4. Check whether there is no error in the "Logs" node of the slot.

**Expect:**
There is no error in the "Logs" node.

**Actual:**
An error shows in the "Logs" node.
![image](https://user-images.githubusercontent.com/45614674/201040705-b406b06c-e0a9-472a-8dd5-61a8f2fa9165.png)

**More Info:**
1. This issue does not reproduce for advanced Function App slots.
2. Refreshing can't fix this issue.

**Comment by nturinski** [comment]
Could you try this again with our current build? I think this [PR](https://github.com/microsoft/vscode-azurefunctions/pull/3426) may have resolved it.

Also, could you try to look at the `Logs` of a slot that already exists? I think that this error may occur for new slots only.

**Comment by v-ruizh** [comment]
@nturinski This issue also reproduces on the latest Function extension build 20221110.36. 
This issue only reproduces in slots of **basic** Function App.

**Comment by nturinski** [comment]
@v-ruizh Yes, but when I created a new basic Function app, and then a slot, I also had that error. But if I waited 15-20 minutes and refreshed it, the error went away.

**Comment by v-ruizh** [comment]
@nturinski Verified this issue on the RC build, it also can be reproduced. The error still exists after refreshing it more than 2 hours later. 

**Correct the information:**
1. This issue reproduces for basic Function App.
2. This issue reproduces for advanced Function App with Consumption app service plan.
3. This issue reproduces for advanced Function App with Windows OS and Premium app service plan.
4. This issue does not reproduce for advanced Function App with App Service Plan or with (Linux OS and Premium app service plan).

**Comment by nturinski** [comment]
Are all of the slots running a valid, working functions project?

**Comment by v-ruizh** [comment]
Yes, the slot running a valid, working function project. This error disappears after deploying the project to the slot.
